src/settings-ui/Settings.UI/ViewModels/PowerLauncherPluginViewModel.cs

```diff
@@ -65,7 +70,7 @@ public bool Disabled
             }
         }
 
-        public bool Enabled => !Disabled;
+        public bool Enabled => !Disabled && _enabledGpoRuleConfiguration != GpoRuleConfigured.Disabled;
```

Comment:
Wouldn’t this be calculated with the prior?

Reply:
@crutkas
Can you please explain a bit more. I didn't understand what you mean with "prior".

And please have in mind that this isn't the final code.